prettier.config.js

```diff
@@ -8,6 +8,16 @@ module.exports = {
 	tabWidth: 2,
 	trailingComma: 'all',
 	useTabs: true,
+	plugins: ['@ianvs/prettier-plugin-sort-imports'],
+	importOrder: [
+		'<BUILT_IN_MODULES>',
+		'<THIRD_PARTY_MODULES>',
+		'',
```

Comment:
what do the blank groups do?

Reply:
looks like it creates an empty line between groups

Reply:
Yeah it creates a newline between groups. Kind of weird, but 🤷